html/pages/assistant.html

```diff
@@ -0,0 +1,245 @@
+								<v-avatar size="32" class="mr-3" color="secondary">
+									<img src="images/favicon/onion.png" alt="Assistant" style="width: 24px; height: 24px;">
+								</v-avatar>
+								<div class="flex-grow-1">
+									<div class="d-flex align-center mb-1">
+										<span class="font-weight-medium text-body-2 mr-2">AI Assistant</span>
+										<span class="text-caption text-medium-emphasis">typing...</span>
+									</div>
+									<div class="text-body-2">
+										<v-progress-linear indeterminate color="secondary" height="2"></v-progress-linear>
+										<span class="text-caption text-medium-emphasis">Thinking...</span>
+									</div>
+								</div>
+							</div>
+						</div>
+					</div>
+				</v-card-text>
+				<v-divider></v-divider>
+				<v-card-actions class="pa-4 d-flex align-center">
+					<v-text-field
+						v-model="newMessage"
+						:placeholder="i18n.chatPlaceholder"
+						variant="outlined"
+						density="compact"
+						hide-details
+						autocomplete="off"
+						spellcheck="false"
+						class="chat-input flex-grow-1 mr-3"
+						@keyup.enter="sendMessage"
+						:disabled="isTyping"
+						data-aid="chat_input"
+					></v-text-field>
+					<v-btn
+						color="primary"
+						:disabled="!newMessage.trim() || isTyping"
+						@click="sendMessage"
+						data-aid="chat_send"
+						class="flex-shrink-0"
+					>
+						<v-icon>{{ isTyping ? 'fa-hourglass-half' : 'fa-paper-plane' }}</v-icon>
+					</v-btn>
+				</v-card-actions>
+			</v-card>
+		</v-col>
+		
+		<!-- Recent Chats Sidebar -->
+		<v-col cols="12" lg="3">
+			<v-card class="chat-history-sidebar">
+				<v-card-title class="py-3 d-flex align-center">
+					<v-icon class="mr-2">fa-history</v-icon>
+					<span class="flex-grow-1">{{ i18n.chatHistory || 'Recent Chats' }}</span>
+					<v-btn icon variant="text" size="x-small" @click="startNewChat" :title="i18n.chatNewChat || 'New Chat'" data-aid="chat_new_button">
+						<v-icon size="small">fa-plus</v-icon>
+					</v-btn>
+				</v-card-title>
+				<v-divider></v-divider>
+				<v-card-text class="chat-history-content pa-0">
+					<v-list v-if="chatHistory.length > 0">
+						<v-list-item
+							v-for="chat in chatHistory.slice().reverse()"
+							:key="chat.id"
+							@click="loadChat(chat)"
+							:class="{ 'bg-primary': chat.id === currentChatId }"
+							data-aid="chat_history_item"
+						>
+							<v-list-item-title class="text-wrap">{{ chat.title }}</v-list-item-title>
+							<v-list-item-subtitle>{{ formatChatDate(chat.lastUpdated) }}</v-list-item-subtitle>
+							<template #append>
+								<v-btn icon variant="text" size="small" @click.stop="deleteChat(chat.id)" data-aid="chat_history_delete">
+									<v-icon size="small" color="error">fa-trash</v-icon>
+								</v-btn>
+							</template>
+						</v-list-item>
+					</v-list>
+					<div v-else class="text-center text-medium-emphasis py-8">
+						<v-icon size="48" class="mb-4">fa-history</v-icon>
```

Comment:
did none of the built-in sizes work? Is that why you had to hardcode a size here? In general I don't want UI sizes (fonts, icons, etc) hardcoded, since it goes against theme compatibility standards.